app/models/lion.rb

```diff
@@ -23,6 +23,17 @@ def self.create_from_image_set(image_set, name)
     lion
   end
 
+  def self.search(search_params)
+    age_params = search_params.extract!(:age_range_start, :age_range_end)
+
+    unless age_params.empty?
+      dob_hash = { date_of_birth: (age_params[:age_range_end]..age_params[:age_range_start]) }
```

Comment:
@iezer In particular, I wasn't sure whether the age_range_end should be the eldest or the youngest age provided by the front-end. What do you think makes the most sense?